packages/ra-data-fakerest/src/index.js

```diff
@@ -92,12 +94,22 @@ export default (data, loggingEnabled = false) => {
                         ...params.data,
                     }),
                 };
+            case UPDATE_MANY:
+                params.ids.forEach(id =>
```

Comment:
Are you sure that Fakerest supports that?

Reply:
The e2e tests confirm it